src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -607,6 +626,38 @@ limitations under the License.
     <attribute code="0x0052" side="server" name="SetpointHoldExpiryTimestamp" type="epoch_s" define="SETPOINT_HOLD_EXPIRY_TIMESTAMP" optional="true" isNullable="true">
       <optionalConform/>
     </attribute>
+    <attribute code="0x0053" side="server" name="MaxThermostatSuggestions" define="MAX_THERMOSTAT_SUGGESTIONS" optional="true" type="int8u" min="5">
```

Comment:
All the attributes should have apiMaturity="provisional", right?

Reply:
done.